homeassistant/components/sensor/fronius.py

```diff
@@ -0,0 +1,157 @@
+'''
+Support for Fronius devices.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/sensor.fronius/
+'''
+import asyncio
+import logging
+from datetime import timedelta
+from aiohttp.client_exceptions import ServerDisconnectedError
+from concurrent.futures._base import TimeoutError
+
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import (CONF_HOST, CONF_SCAN_INTERVAL)
+from homeassistant.helpers.event import async_track_time_interval
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+from homeassistant.helpers.entity import Entity
```

Comment:
Please keep the imports ordered. Use `isort` if you don't want to do it manually.